app/src/main/java/com/pr0gramm/app/services/UserService.kt

```diff
@@ -351,6 +351,13 @@ class UserService(private val api: Api,
         return Graph(start.millis.toDouble(), now.millis.toDouble(), points)
     }
 
+    fun loadBenis(userId:Int = loginState.id):Observable<BenisRecord>{
+        val date = Instant(0)
```

Comment:
Nein, die Funktion soll nur direkt ohne zu blockieren ein Observable zurück geben. Erst wenn man zu dem Observable subscribed, erst dann wird die Anfrage (potentiell in einem anderen Thread)  ausgeführt.